modules/ws/src/test/java/org/mule/module/ws/consumer/WSConsumerWsdlProxyTestCase.java

```diff
@@ -25,7 +25,7 @@
 import org.mule.tck.junit4.rule.DynamicPort;
 import org.mule.util.IOUtils;
 
-import com.google.common.net.MediaType;
+import static com.google.common.net.MediaType.APPLICATION_XML_UTF_8;;
```

Comment:
static imports are declared before other imports. Change the order in the ide configuration